lib/matplotlib/backends/backend_wx.py

```diff
@@ -1614,10 +1614,12 @@ def press(self, event):
             if not self.retinaFix:
                 self.wxoverlay = wx.Overlay()
             else:
-                self.savedRetinaImage = self.canvas.copy_from_bbox(
-                    self.canvas.figure.gca().bbox)
-                self.zoomStartX = event.xdata
-                self.zoomStartY = event.ydata
+                if event.inaxes is not None:
+                    self.savedRetinaImage = self.canvas.copy_from_bbox(
+                        event.inaxes.bbox)
+                    self.zoomStartX = event.xdata
+                    self.zoomStartY = event.ydata
+                    self.zoomAxes = event.inaxes
```

Comment:
Thats easy enough to do. However, making it private makes it inconsistent with the zoomStart variables, which is why I left it as is. I could also change those, but wasn't sure if they were public for a reason. Thoughts?